Source/Particles/NamedComponentParticleContainer.H

```diff
@@ -0,0 +1,128 @@
+#ifndef NamedComponentParticleContainer_H_
+#define NamedComponentParticleContainer_H_
+
+#include <AMReX.H>
+#include <AMReX_AmrParGDB.H>
+#include <AMReX_Particles.H>
+#include "Utils/TextMsg.H"
```

Comment:
- include WarpX includes before AMReX
- Includes missing for classes used from stdlib